### PR TITLE
Fix multi-node scale down callout

### PR DESCRIPTION
**Pull request by gemma-shay**
https://cockroachlabs.atlassian.net/browse/DOC-2289
For scaling a multi-node cluster down to a single-node cluster:
Change “if you need to do this, contact support” to “if you need to do this, take a backup and restore into a new single node cluster.”

### CODE REVIEW

File: cockroachcloud/cluster-management.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ CockroachDB Cloud contract customers cannot scale clusters through the Console.
 {{site.data.alerts.end}}
 
 {{site.data.alerts.callout_info}}
-At this time, you cannot use the Console to scale a multi-node cluster down to a single-node cluster. If you need to do this, contact [Support](https://support.cockroachlabs.com).
+You cannot scale a multi-node cluster down to a single-node cluster. If you need to do this, [backup](run-bulk-operations.html?filters=cloud#backup-and-restore-data) your cluster and [restore](run-bulk-operations.html?filters=cloud#restore-a-cluster) it into a single-node cluster.
```
Comment (chrisseto) [inline cockroachcloud/cluster-management.md:64]: Possible wording nit: Would saying "_new_ single-node cluster" be more or less clear?

**Review by chrisseto — APPROVED** [review]
LGTM! Thanks for jumping on this so fast.

File: cockroachcloud/cluster-management.md (hunk 1 above)
Comment (gemma-shay) [reply cockroachcloud/cluster-management.md]: good nit!

File: cockroachcloud/cluster-management.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ CockroachDB Cloud contract customers cannot scale clusters through the Console.
 {{site.data.alerts.end}}
 
 {{site.data.alerts.callout_info}}
-At this time, you cannot use the Console to scale a multi-node cluster down to a single-node cluster. If you need to do this, contact [Support](https://support.cockroachlabs.com).
+You cannot scale a multi-node cluster down to a single-node cluster. If you need to do this, [backup](run-bulk-operations.html?filters=cloud#backup-and-restore-data) your cluster and [restore](run-bulk-operations.html?filters=cloud#restore-a-cluster) it into a new single-node cluster.
```
Comment (ianjevans) [inline cockroachcloud/cluster-management.md:64]:
```suggestion
You cannot scale a multi-node cluster down to a single-node cluster. If you need to scale down to a single-node cluster, [backup](run-bulk-operations.html?filters=cloud#backup-and-restore-data) your cluster and [restore](run-bulk-operations.html?filters=cloud#restore-a-cluster) it into a new single-node cluster.
```

**Review by ianjevans — APPROVED** [review]
Just a suggested wording nit.

**Comment by emily-horing** [comment]
@gemma-shay we also have a line in our [FAQ](https://www.cockroachlabs.com/docs/cockroachcloud/frequently-asked-questions.html#how-do-i-add-nodes) that incorrectly states we cannot scale up from a 1 node cluster. Can we fix that too? Maybe in a new PR

**Pull request merged**